_events/sr2019/london-kickstart.md

```diff
@@ -0,0 +1,57 @@
+---
+title: London Kickstart
+date: 2018-11-10 10:00:00
+layout: event
+type: kickstart
+location: Thread, Whitechapel
+---
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams.
+
+**There is limited space at the London Kickstart, please only attend the London
+Kickstart if we have informed you that you have a place there.**
+
+If you are unable to attend Kickstart, please [let us know][teams-contact] so we
+can arrange another way to get you your kit.
+
+**Please bring your laptops!**
+
+## Directions
+
+The London Kickstart is being held in the [offices of Thread][venue-map] at:
+
+1 Alie Street,  
+Whitechapel,  
+London,  
+E1 8DE  
+
+The easiest way to travel will be by public transport -- Aldgate, Aldgate East
+and Tower Gateway stations are all within a five minute walk, and Liverpool
+Street station is a ten minute walk away. There are also a large number of
+busses which stop nearby.
+
+<!-- TODO: find out whether we can provide parking
```

Comment:
Has this TODO been done?

Reply:
Sort of (see https://github.com/srobo/website/pull/91#issuecomment-429630409 for details).
I believe we would be able to provide parking, I don't know what the vehicle size limit would be. I'll check that on Monday, but I suspect anyone who really wants to drive would ask us about it.